api/api.proto

```diff
@@ -3155,11 +3155,13 @@ message SdkStorageRebalanceRequest {
   repeated  StorageRebalanceTriggerThreshold trigger_thresholds = 1;
   // TrialRun if true the job only produces steps that would be taken without making any changes
   bool trial_run = 2;
-  // PoolSelectorLabels allows selecting pools to which trigger thresholds will apply
-  map<string, string> pool_selector_labels = 3;
+  // SourcePoolSelectorLabels allows selecting pools to which trigger thresholds will apply as source
+  map<string, string> source_pool_selector_labels = 3;
+  // TargetPoolSelectorLabels allows selecting pools to which trigger thresholds will apply as target
+  map<string, string> target_pool_selector_labels = 4;
```

Comment:
@lpabon heads up.

We are changing fields in the proto without updating SDK version as this is the rebalance API and this has not been released yet. These changes incorporate feedback during 2.6.0 testing.